specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2021-02-01-preview/examples/RedisEnterpriseDatabasesForceUnlink.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parameters": {
+    "databaseName": "default",
+    "clusterName": "cache1",
+    "resourceGroupName": "rg1",
+    "api-version": "2021-02-01-preview",
+    "subscriptionId": "subid",
+    "parameters": {
+      "ids": [
+        "/subscriptions/subid2/resourceGroups/rg2/providers/Microsoft.Cache/redisEnterprise/cache2/databases/default"
+      ]
+    }
+  },
+  "responses": {
+    "200": {},
+    "202": {
+      "headers": {
+        "Azure-AsyncOperation": "https://management.azure.com/subscriptions/subid/providers/Microsoft.Cache/...pathToOperationStatus...",
+        "Location": "https://management.azure.com/subscriptions/subid/providers/Microsoft.Cache/...pathToOperationResult..."
```

Comment:
Why are both Location & Azure-AsyncOperation headers needed? As per schema spec, the final operation was supposed to be be through Azure-AsyncOperation